### PR TITLE
Switch to using 8.0.0 API for examples

### PR DESCRIPTION
**Pull request by Chris-Sigopt**
Update the examples to use training runs, based on the guidance in the docs. 

These are closer to the docs but not really runnable, but then again the previous stuff wasn't either.

### CODE REVIEW

File: examples/basic.py [hunk 1]
```diff
@@ -15,22 +17,28 @@ def evaluate_metric(assignments):
 
   connection = Connection(client_token=the_args.client_token, _show_deprecation_warning=False)
 
+  
+  sigopt.log_dataset("Example dataset") #Descriptor of what kind of dataset you are modeling
+  sigopt.log_metadata(key="Dataset Source", value="Example Source") #Useful for keeping track of where you got the data 
+  sigopt.log_metadata(key="Feature Pipeline Name", value="Example Pipeline") #e.g. Sklern, xgboost, etc.
+  sigopt.log_model("Example Model Technique") # What kind of learning you are attemping
   # Create an experiment with one paramter, x
```
Comment (dwanderson-intel) [inline examples/basic.py:25]: consistent formatting: do we want inline comments or above-comments? and there should be a space between `#` and the first character (lingering todo item to implement linting / auto-formatting on this repo)

File: examples/basic.py [hunk 2]
```diff
@@ -15,22 +17,28 @@ def evaluate_metric(assignments):
...
-  experiment = connection.experiments().create(
+  experiment = sigopt.create_experiment(
     name="Basic Test experiment",
     project="sigopt-examples",
+    type="offline",
```
Comment (dwanderson-intel) [inline examples/basic.py:29]: pretty sure this is the default, in which case we prefer not to include this since the name is a bit legacy

**Review by dwanderson-intel — COMMENTED** [review]
do we need / can we got Product input on what we want this basic example to look like? if they've signed off then we're good to go

File: examples/parallel.py [hunk 3]
```diff
@@ -1,63 +1,45 @@
...
-  parser.add_argument('--runner_count', type=int, default=2)
+  parser.add_argument('--observation_budget', type=int, default=20)
   parser.add_argument('--client_token', required=True, help="Find your CLIENT_TOKEN at https://sigopt.com/tokens")
   the_args = parser.parse_args()
 
-  client_token = the_args.client_token
-  conn = Connection(client_token=client_token)
-  experiment = conn.experiments().create(
-    name="Parallel Test Franke Function",
+  connection = Connection(client_token=the_args.client_token, _show_deprecation_warning=False)
```
Comment (tjs-intel) [inline examples/parallel.py:18]: remove this line please

**Comment by tjs-intel** [comment]
Echoing Dan's comment about running this by product

File: examples/parallel.py [hunk 4]
```diff
@@ -1,63 +1,45 @@
...
     project="sigopt-examples",
-    parameters=[
-      {'name': 'x1', 'bounds': {'max': 1.0, 'min': 0.0}, 'type': 'double'},
-      {'name': 'x2', 'bounds': {'max': 1.0, 'min': 0.0}, 'type': 'double'},
-    ],
+    type="offline",
+    parameters=[{'name': 'x', 'bounds': {'max': 50.0, 'min': 0.0}, 'type': 'double'}],
+    metrics=[{"name":"holdout_accuracy", "objective":"maximize"}],
+    parallel_bandwidth=3, #number of machines you are running learning on 
+    observation_budget=the_args.observation_budget,
```
Comment (meghanaravikumar) [inline examples/parallel.py:33]: @Chris-Sigopt this example rewrite looks good. Couple of comments: we no longer use "observation" or "suggestion" externally (and really should move on internally). Could you change the parser argument naming to "budget" or "experiment budget"? The sigopt.create_experiment() call should also be taking in a `budget` and not an `observation_budget`. other than that this looks good

**Review by meghanaravikumar — CHANGES_REQUESTED** [review]

File: examples/parallel.py (hunk 4 above)
Comment (Chris-Sigopt) [reply examples/parallel.py:33]: Based off of https://app.sigopt.com/docs/experiments/options it looks like "budget" is the right word, great catch! Will correct that now.

**Review by dwanderson-intel — APPROVED** [review]

**Pull request merged**